src/main/java/edu/cornell/kfs/concur/dto/ConcurStandardAccountingExtractDTO.java

```diff
@@ -0,0 +1,6 @@
+package edu.cornell.kfs.concur.dto;
+
+public class ConcurStandardAccountingExtractDTO {
```

Comment:
Since this object won't be persisted, I didn't want to include in the business object package. The expected functionality is more just a POJO holding object rather than a DTO, but the DTO name didn't feel all that off.